src/model/key_data/user_id.py

```diff
@@ -0,0 +1,8 @@
+import dataclasses
+
+@dataclasses.dataclass
+class UserId:
```

Comment:
```suggestion
class KeyUserId:
```

Discordのuser_idと混同する